src/main/java/net/minecraftforge/gradle/user/ReobfTaskFactory.java

```diff
@@ -196,15 +220,26 @@ public void extraFiles(Object... o)
                 reobf.addSecondarySrgFile(obj);
             }
         }
-        
+
+        @Deprecated
+        @Override
         public void useSrgSrg()
         {
-            reobf.setPrimarySrg(plugin.delayedFile(Constants.SRG_MCP_TO_SRG));
+            setMappingType(ReobfMappingType.SEARGE);
+            warnDeprecation("useSrgSrg()", "mappingType");
         }
 
+        @Deprecated
+        @Override
         public void useNotchSrg()
         {
-            reobf.setPrimarySrg(plugin.delayedFile(Constants.SRG_MCP_TO_NOTCH));
+            setMappingType(ReobfMappingType.NOTCH);
+            warnDeprecation("useNotchSrg()", "mappingType");
+        }
+
+        private void warnDeprecation(String old, String new_)
```

Comment:
Perhaps this method should be moved to a more common area (`BasePlugin`?) so that it can be reused? If so, just be more clear in the passed parameters.